web/src/scroll_util.ts

```diff
@@ -58,16 +65,19 @@ export function scroll_delta(opts) {
     return delta;
 }
 
-export function scroll_element_into_container($elem, $container, sticky_header_height = 0) {
+export function scroll_element_into_container(
+    $elem: JQuery,
+    $container: JQuery,
+    sticky_header_height = 0,
+): void {
     // This does the minimum amount of scrolling that is needed to make
     // the element visible.  It doesn't try to center the element, so
     // this will be non-intrusive to users when they already have
     // the element visible.
-
     $container = get_scroll_element($container);
     const elem_top = $elem.position().top - sticky_header_height;
-    const elem_bottom = elem_top + $elem.innerHeight();
-    const container_height = $container.height() - sticky_header_height;
+    const elem_bottom = elem_top + ($elem.innerHeight() ?? 0);
```

Comment:
Does TypeScript require typecasting for methods like `height()`? I was under the impression that these methods could only return a number.

Reply:
It turns out these methods can return undefined as well when your JQuery set has no elements.